cfme/tests/cloud_infra_common/test_discovery.py

```diff
@@ -54,6 +54,7 @@ def wait_for_vm_state_changes(vm, timeout=600):
         raise CFMEException("VM should be Archived but it is Orphaned now.")
 
 
+@pytest.mark.rhv1
```

Comment:
I general this most it tier 2. I've put it in tier 1 since we had several bugs around this.
But on a second thought, a bug here can be also discovered in tier 2. I'll move it to tier 2.

Reply:
Will move it to T2.